src/main/java/org/skriptlang/skript/bukkit/input/elements/conditions/CondIsPressingKey.java

```diff
@@ -0,0 +1,96 @@
+package org.skriptlang.skript.bukkit.input.elements.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Input;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerInputEvent;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.input.InputKey;
+
+@Name("Is Pressing Key")
+@Description("Checks if a player is pressing a certain input key.")
+@Examples({
+	"on player input:",
+	"\tif player is pressing forward movement key:",
+		"\t\tsend \"You are moving forward!\""
+})
+@Since("INSERT VERSION")
+@Keywords({"press", "input"})
+@RequiredPlugins("Minecraft 1.21.2+")
+public class CondIsPressingKey extends Condition {
+
+	static {
+		if (Skript.classExists("org.bukkit.event.player.PlayerInputEvent")) {
```

Comment:
When was this added? If it's a 1.21.2/1.21.3 difference I don't think we need to be concerned about this check - we do not support 1.21.2

Reply:
this was 1.12.2 iirc

Reply:
> When was this added? If it's a 1.21.2/1.21.3 difference I don't think we need to be concerned about this check - we do not support 1.21.2

Well if it's there and it works why not just keep it :sob: why are we going out of our way not to support it

Reply:
I think it makes it unnecessarily messier, but I'll leave it up to you 🙂

Reply:
the one person using this on 1.21.2 will thank ya